db/migrate/20191017185340_add_unique_constraint_to_profile_by_account.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniqueConstraintToProfileByAccount < ActiveRecord::Migration[5.2]
+  def change
+    add_index(:profiles , %i[account_id name], unique: true)
```

Comment:
This will very quickly need to be:

```suggestion
    add_index(:profiles , %i[account_id name benchmark_id], unique: true)
```